app/assets/stylesheets/govuk_publishing_components/components/_document-list.scss

```diff
@@ -103,5 +103,66 @@
 .gem-c-document-list__highlight-text {
   @include govuk-font(16, bold);
   margin: 0 0 govuk-spacing(3) 0;
+}
+
+.gem-c-document-list__children {
+  margin-bottom: 0;
+
+  @include govuk-media-query($from: desktop) {
+    margin-left: govuk-spacing(4);
+    margin-top: govuk-spacing(4);
+
+    @supports (display: grid) {
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      column-gap: govuk-spacing(3);
+    }
+  }
+}
+
+.gem-c-document-list-child {
+  @include govuk-font($size: 16);
+
+  @include govuk-media-query($from: desktop) {
+    padding-bottom: govuk-spacing(2);
+  }
+
+  @include govuk-media-query($until: desktop) {
```

Comment:
We seem to have one media query for until: desktop and one for from: desktop here. Do we need both?

Reply:
I've rewritten it to only have one, but because of the need to override stuff I don't think it actually saves any lines of code. Hopefully easier to read though.